### DISCUSSION TITLE
GitHub Pages for all private repos become 404 after Pro subscription expires. Then it has to be reenabled per repo after renewing Pro subscription

### DISCUSSION THREAD

**Discussion by ZufengW** [community/community#59800] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Recently someone told me that the GitHub pages for one of my repos had started showing a 404 page. I looked into this. It was confusing because there were no emails or notifications on GitHub explaining the change. I noticed that the GitHub pages for some of my repos still worked, but others didn't. Eventually I realized it was because all the ones that didn't work were private repos, and my GitHub Pro subscription had expired recently.

So I subscribed to GitHub Pro again.

But GitHub pages still didn't work for those private repos. I realized it was because the GitHub Pages settings had **reset** for all my private repos:

![Screen Shot 2023-07-01_github_pages_settings_reset](https://github.com/community/community/assets/15045948/9edb61e2-04c3-49fe-bab6-57494e6ca3d2)

So I had to go through every single private repo and manually reconfigure the GitHub pages settings. This was inconvenient. Maybe it would be better if it saved the settings or if the pages didn't become 404 at the end of a Pro subscription.

My issue is similar to https://github.com/orgs/community/discussions/54528 but with the extra annoyance of having to reenable GitHub pages on multiple Private repos after subscribing to Pro.

**Comment by queenofcorgis** [comment]
Hey there @ZufengW, that does sound incredibly frustrating and manual. I'm going to explore updating our documentation so this is better communicated and expected. Really appreciate you taking the time to leave this feedback :octocat:.
